kyuubi-common/src/main/scala/org/apache/kyuubi/operation/log/OperationLog.scala

```diff
@@ -195,6 +195,8 @@ class OperationLog(path: Path) {
   }
 
   def close(): Unit = synchronized {
+    if (!initialized) return
```

Comment:
When the operation log is not initialized, the close method will throw an exception.

```java
Caused by: java.nio.file.NoSuchFileException:

        at org.apache.kyuubi.operation.log.OperationLog.reader$lzycompute(OperationLog.scala:89)
        at org.apache.kyuubi.operation.log.OperationLog.reader(OperationLog.scala:89)
        at org.apache.kyuubi.operation.log.OperationLog.$anonfun$close$1(OperationLog.scala:201)
        at org.apache.kyuubi.operation.log.OperationLog.trySafely(OperationLog.scala:221)
```